src/Native/System.Native/pal_networkchange.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma once
+
+#include "pal_types.h"
+#include <linux/netlink.h>
```

Comment:
I don't believe this header is included in any cpp files on Linux. pal_networkchange.cpp includes it in a conditional section.